README.md

```diff
@@ -173,124 +173,7 @@ versioning in the future. But that decision is yet to be made.
-- [ING Bank](https://www.ing.com)
-- [Inner Product](https://inner-product.com)
-- [innFactory GmbH](https://www.innfactory.de/)
-- [Intent HQ](https://www.intenthq.com/)
-- [iofod](https://www.iofod.com/)
-- [Iterators](https://www.iteratorshq.com/)
-- [ITV](https://www.itv.com/)
-- [Kaluza](https://www.kaluza.com)
-- [Lightform](https://lightform.com/)
-- [Lookout](https://www.lookout.com)
-- [Medidata](https://www.medidata.com/)
-- [Merit](https://merits.com)
-- [Metacommerce](https://www.metacommerce.ru)
-- [MYOB](https://www.myob.com)
-- [NCR Edinburgh](https://ncredinburgh.com/)
-- [Netflix](https://jobs.netflix.com)
-- [NetworkedAssets](https://www.networkedassets.com/)
-- [Nezasa](https://www.nezasa.com)
-- [Ocado Technology](https://ocadotechnology.com)
-- [On Air Entertainment](https://onairentertainment.com/)
-- [OVO Energy](https://ovoenergy.com)
-- [Packlink](https://packlink.com/)
-- [Permutive](https://permutive.com/)
-- [Precog](https://precog.com)
-- [Raiffeisenbank Russia](https://www.raiffeisen.ru/)
-- [Rakuten](https://www.rakuten.com)
-- [RBC Capital Markets](https://www.rbccm.com)
-- [REA Group](https://www.realestate.com.au/)
-- [Reality Games](https://wearerealitygames.com)
-- [Revonte](https://revonte.com)
-- [Rewards Network](https://rewardsnetwork.com)
-- [RMS](https://www.rms.com/)
-- [Rudder](https://rudder.io)
-- [Scala Center](https://scala.epfl.ch)
-- [Scalac](https://scalac.io)
-- [SecurityScorecard](https://securityscorecard.io/)
-- [SEEK Limited](https://www.seek.com.au/)
-- [Snowplow Analytics](https://snowplowanalytics.com/)
-- [SoftwareMill](https://softwaremill.com)
-- [Spiceworks](https://www.spiceworks.com/)
-- [Spotahome](https://spotahome.com)
-- [Spotify](https://www.spotify.com)
-- [SpotX](https://www.spotx.tv/)
-- [SpringerNature](https://www.springernature.com)
-- [SRF](https://www.srf.ch)
-- [Standard Chartered](https://www.sc.com)
-- [Stripe](https://stripe.com)
-- [SWCoE at Philips](https://www.philips.com/)
-- [SWOP](https://swop.cx/)
-- [Tapad](https://www.tapad.com/)
-- [Tecsisa](https://www.tecsisa.com)
-- [Teikametrics](http://teikametrics.com)
-- [Tenable](https://www.tenable.com)
-- [The Guardian](https://www.theguardian.com)
-- [Tinkoff](https://tinkoff.ru)
-- [Tranzzo](https://tranzzo.com)
-- [Underscore Consulting](https://underscore.io/)
-- [vidIQ](https://vidiq.com/)
-- [Waves Platform](https://wavesplatform.com/)
-- [Wegtam GmbH](https://www.wegtam.com)
-- [Wejo](https://www.wejo.com/)
-- [WeWork](https://www.wework.com)
-- [Wix.com](https://www.wix.com)
-- [Wolt](https://wolt.com/)
-- [XITE](https://xite.com/)
-- [Zalando](https://zalando.com)
-- [Zendesk](https://www.zendesk.com)
+A (non-exhaustive) list of companies that use Cats in production is featured in [ADOPTERS.md](./ADOPTERS.md). Don't see yours? [You can add it in a PR!](https://github.com/typelevel/cats/edit/main/ADOPTERS.md) And if you can, consider [supporting us](https://opencollective.com/typelevel).
```

Comment:
Is this `[ADOPTERS.md](./ADOPTERS.md)` the reason we are duplicating the readme? I think it would be fine to link to the website for both. It would have to be be an absolute URL.

But I'm also favor of the readme and index eventually becoming separate anyway, so might as well happen now.

Reply:
I also deleted the 'documentation' paragraph in the index page source. So they differ not only by the link already :)